wsapi.go

```diff
@@ -682,11 +682,11 @@ type voiceChannelJoinOp struct {
 
 // ChannelVoiceJoin joins the session user to a voice channel.
 //
-//    gID     : Guild ID of the channel to join.
-//    cID     : Channel ID of the channel to join.
-//    mute    : If true, you will be set to muted upon joining.
-//    deaf    : If true, you will be set to deafened upon joining.
-func (s *Session) ChannelVoiceJoin(gID, cID string, mute, deaf bool) (voice *VoiceConnection, err error) {
+//	gID     : Guild ID of the channel to join.
+//	cID     : Channel ID of the channel to join.
+//	mute    : If true, you will be set to muted upon joining.
+//	deaf    : If true, you will be set to deafened upon joining.
+func (s *Session) ChannelVoiceJoin(gID, cID string, mute, deaf bool, h VoiceSpeakingUpdateHandler) (voice *VoiceConnection, err error) {
```

Comment:
suggestion(non-blocking): To align with the pattern of supporting multiple handlers, this should probably accept an array of handlers or perhaps be an alternative join function

Eg: (support multiple)
```suggestion
func (s *Session) ChannelVoiceJoin(gID, cID string, mute, deaf bool, handlers []VoiceSpeakingUpdateHandler) (voice *VoiceConnection, err error) {
```

Eg. (alternate join function)
```suggestion
func (s *Session) ChannelVoiceJoin(gID, cID string, mute deaf bool) { ChannelVoiceJoinHandlers(gID, cID, mute, deaf, []VoiceSpeakingUpdateHandler{}) }
func (s *Session) ChannelVoiceJoinHandlers(gID, cID string, mute, deaf bool, handlers []VoiceSpeakingUpdateHandler) (voice *VoiceConnection, err error) {
```